src/main/kotlin/org/rust/lang/core/resolve/Processors.kt

```diff
@@ -81,6 +81,22 @@ fun collectResolveVariants(referenceName: String, f: (RsResolveProcessor) -> Uni
     return result
 }
 
+fun <T : ScopeEntry> collectResolveVariantsAsSEs(referenceName: String, f: ((T) -> Boolean) -> Unit): List<T> {
```

Comment:
Maybe rename it somehow? because at this moment the name is rather unclear
Also, it would be great to have some docs about why when it can be used